UET/Redpoint.Uet.BuildPipeline.Executors.Jenkins/JenkinsBuildExecutor.cs

```diff
@@ -1,8 +1,9 @@
 ﻿namespace Redpoint.Uet.BuildPipeline.Executors.Jenkins
 {
     using Microsoft.Extensions.Logging;
-    using Redpoint.Uet.BuildPipeline.Executors;
-    using Redpoint.Uet.BuildPipeline.Executors.BuildServer;
+    using Executors;
+    using BuildServer;
+    using Concurrency;
```

Comment:
Prefer full namespace `using` directives (if this was caused by an Intellisense auto-correct, we should update `.editorconfig` to avoid it).